config/webpacker.yml

```diff
@@ -36,7 +36,6 @@ default: &default
   extensions:
     - .tsx
     - .ts
-    - .jsx
```

Comment:
nice.  i guess this will prevent us from ever writing jsx for something other than our react code.